resin/models/tag.py

```diff
@@ -0,0 +1,308 @@
+
+        Examples:
+            >>> resin.models.tag.application.get_all_by_application('1005767')
+            [{u'application': {u'__deferred': {u'uri': u'/resin/application(1005767)'}, u'__id': 1005767}, u'tag_key': u'appTa1', u'id': 12887, u'value': u'Python SDK', u'__metadata': {u'type': u'', u'uri': u'/resin/application_tag(12887)'}}, {u'application': {u'__deferred': {u'uri': u'/resin/application(1005767)'}, u'__id': 1005767}, u'tag_key': u'appTag2', u'id': 12888, u'value': u'Python SDK', u'__metadata': {u'type': u'', u'uri': u'/resin/application_tag(12888)'}}]
+
+
+        """
+
+        params = {
+            'filter': 'application',
+            'eq': app_id
+        }
+
+        return super(ApplicationTag, self).get_all(params=params)
+
+    def get_all(self):
+        """
+        Get all application tags.
+
+        Returns:
+            list: list contains application tags.
+
+        Examples:
+            >>> resin.models.tag.application.get_all()
+            [{u'application': {u'__deferred': {u'uri': u'/resin/application(1005160)'}, u'__id': 1005160}, u'tag_key': u'appTag', u'id': 12886, u'value': u'Python SDK', u'__metadata': {u'type': u'', u'uri': u'/resin/application_tag(12886)'}}, {u'application': {u'__deferred': {u'uri': u'/resin/application(1005767)'}, u'__id': 1005767}, u'tag_key': u'appTa1', u'id': 12887, u'value': u'Python SDK', u'__metadata': {u'type': u'', u'uri': u'/resin/application_tag(12887)'}}, {u'application': {u'__deferred': {u'uri': u'/resin/application(1005767)'}, u'__id': 1005767}, u'tag_key': u'appTag2', u'id': 12888, u'value': u'Python SDK', u'__metadata': {u'type': u'', u'uri': u'/resin/application_tag(12888)'}}]
+
+        """
+
+        return super(ApplicationTag, self).get_all()
+
+    def set(self, app_id, tag_key, value):
+        """
+        Set an application tag (update tag value if it exists).
+
+        Args:
+            app_id (str): application id.
+            tag_key (str): tag key.
+            value (str): tag value.
+
+        Returns:
+            dict: dict contains application tag info if tag doesn't exist.
+            OK: if tag exists.
+
+        Examples:
+            >>> resin.models.tag.application.set('1005767', 'tag1','Python SDK')
+            {u'application': {u'__deferred': {u'uri': u'/resin/application(1005767)'}, u'__id': 1005767}, u'tag_key': u'tag1', u'id': 12889, u'value': u'Python SDK', u'__metadata': {u'type': u'', u'uri': u'/resin/application_tag(12889)'}}
+            >>> resin.models.tag.application.set('1005767', 'tag1','Resin Python SDK')
+            OK
+
+        """
+
+        return super(ApplicationTag, self).set(app_id, tag_key, value)
+
+    def remove(self, app_id, tag_key):
+        """
+        Remove an application tag.
+
+        Args:
+            app_id (str): application id.
+            tag_key (str): tag key.
+
+        Examples:
+            >>> resin.models.tag.application.set('1005767', 'tag1')
```

Comment:
The example here should be updated to demonstrate `remove`.

Reply:
Updated.